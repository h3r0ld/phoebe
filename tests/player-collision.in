start map.txt
hardworking-little-robot-random off
move E
end-turn
quit
