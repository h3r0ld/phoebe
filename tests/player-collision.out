Player 0: Cell(22,8) Speed: 2 CanChangeDirection: true
Player 0: New Position: Cell(22, 10)
Player 0: Distance - 0; Player 1: Distance - 0; Winner: Player 0
